src/ast/postprocessing/structured_documentation_reconstruction.ts

```diff
@@ -251,9 +255,14 @@ export class StructuredDocumentationReconstructingPostprocessor
 
         /**
          * Dangling structured documentation can only be located in statements,
-         * that may have nested children (`StatementWithChildren` or `ContractDefinition`).
```

Comment:
Comment is a little out of date. Should be "Dangling structured documentation is currently can currently be added to Statements, ContractDefinitions, EnumDefinitions and StructDefinitions"